From 8405ffda003d1d2616907f53cb6449fef222d38f Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Wed, 3 Oct 2018 20:33:54 -0700
Subject: [PATCH 10/11] DO-NOT-UPSTREAM: KEYS: Disable -Wvarargs for trusted.o

Documented with no clear way to solve it, hide it for now.

Link: https://github.com/ClangBuiltLinux/linux/issues/41
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 security/keys/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/security/keys/Makefile b/security/keys/Makefile
index ef1581b337a3..0cde086e3eac 100644
--- a/security/keys/Makefile
+++ b/security/keys/Makefile
@@ -29,3 +29,5 @@ obj-$(CONFIG_KEY_DH_OPERATIONS) += dh.o
 obj-$(CONFIG_BIG_KEYS) += big_key.o
 obj-$(CONFIG_TRUSTED_KEYS) += trusted.o
 obj-$(CONFIG_ENCRYPTED_KEYS) += encrypted-keys/
+
+CFLAGS_trusted.o += $(call cc-disable-warning, varargs)
-- 
2.19.0

